src/locales/en.json

```diff
@@ -971,6 +1021,7 @@
       "type": {
         "AWS": "AWS",
         "Azure": "Microsoft Azure",
+        "GCP": "GCP",
```

Comment:
My first thought is that this should not be abbreviated. However, this looks like more dead code.

Except for onboarding.type_options, the rest is dead code. We no longer have an onboarding modal.